index.js

```diff
@@ -1,62 +1,35 @@
 const express = require('express');
-const cors = require('cors');
-const { ApolloServer } = require('apollo-server-express');
-const jwt = require('jsonwebtoken');
+const { connectDatabase } = require('./src/database');
 const { passport } = require('./src/auth');
-const db = require('./src/database');
-
-const PORT = process.env.PORT || 5000;
-
+const { buildApolloServer } = require('./src/graphql');
 const { register: registerRoutes } = require('./src/routes');
+const { initializeCors } = require('./src/cors');
+const { generateJwt } = require('./src/token');
 
-const typeDefs = require('./src/graphql/typeDefs');
-const resolvers = require('./src/graphql/resolvers');
-
-const server = new ApolloServer({
-    typeDefs,
-    resolvers,
-    introspection: true,
-    playground: true,
-});
+const PORT = process.env.PORT || 5000;
 
 const app = express();
 
-app.use(express.static('public'));
+connectDatabase();
```

Comment:
Database now connects before the auth routes get defined, which requires the `User` model.